README.md

```diff
@@ -19,7 +19,7 @@ A full history of the different versions of Diff can be found in the [release no
 
 **Diff 3.x:**
 
-* PHP 7.2 or later (tested with PHP 7.2 up to PHP 8.1)
+* PHP 7.4 or later (tested with PHP 7.4 up to PHP 8.4)
```

Comment:
The header is about `3.x`, not `latest 3.x`. So only the upper bound should be increased

Reply:
Fair. But since we are only testing now with PHP 7.4+, I would still change the lower bound for 'tested with'.